docs/griptape-framework/structures/workflows.md

```diff
@@ -7,7 +7,8 @@ A [Workflow](../../reference/griptape/structures/workflow.md) is a non-sequentia
 
 Workflows have access to the following [context](../../reference/griptape/structures/workflow.md#griptape.structures.workflow.Workflow.context) variables in addition to the [base context](./tasks.md#context):
 
-* `parent_outputs`: outputs into the current task referenceable by parent task IDs.
+* `parent_outputs`: dictionary containing mapping of parent ids to their outputs.
+* `parents_output`: string containing the concatenated outputs of all parent tasks.
```

Comment:
Do we still want to be able to refer to parent outputs by index? If so, want to add something like a `parent_outputs_list` as a part of this change too?

Reply:
We can still index parents and to find outputs directly, right? Something like `parents[2].output`.

Reply:
Yes ^

Reply:
Awesome, I'll try to remember this the next time it comes up!